src/components/HomePage/SearchResultsArea/Offer/useChipFieldSearchOffer.spec.js

```diff
@@ -0,0 +1,160 @@
+import { createTheme } from "@material-ui/core";
+import React from "react";
+import { MemoryRouter } from "react-router-dom";
+import { renderWithStoreAndTheme, fireEvent } from "../../../../test-utils";
+import Offer from "./Offer";
+import OfferWidget from "./OfferWidget";
+import useChipsFieldSearch from "./useChipsFieldSearch";
+
+jest.mock("./useChipsFieldSearch");
+const mockHistoryPush = jest.fn();
+
+jest.mock("react-router-dom", () => ({
+    ...jest.requireActual("react-router-dom"),
+    useHistory: () => ({
+        push: mockHistoryPush,
```

Comment:
Is this working? Asking because when I implemented the URL filters thing u could not get this mock to work.

Reply:
The tests seem to be working, so I'm guessing it works :rocket: :tada: